stock_inventory_revaluation/migrations/9.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# © 2016 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, SUPERUSER_ID
+from openupgradelib import openupgrade
+
+
+def migrate_inventory_revaluation(env):
+    revaluations = env['stock.inventory.revaluation'].search([])
+    for revaluation in revaluations:
+        product_variants = revaluation.product_template_id.\
+            product_variant_ids.mapped('id')
+        if revaluation.product_template_id and\
+                revaluation.product_template_id.product_variant_count > 1:
+            for product_id in product_variants[1:]:
+                new_revaluation = revaluation.\
+                    copy(default={'product_id': product_id,
+                                  'state': 'posted',
+                                  'account_move_id': revaluation.
+                                  account_move_id.id})
+                quants = env['stock.inventory.revaluation.quant'].\
+                    search([('product_id', '=', product_id),
+                            ('revaluation_id', '=', revaluation.id)])
+                if quants:
+                    quants.write({'revaluation_id': new_revaluation.id})
+        prod_dict = {'product_id': product_variants[0]}
```

Comment:
During a migration I get this error:
`File "/opt/odoo9/parts/stock-logistics-warehouse9/stock_inventory_revaluation/migrations/9.0.1.0.0/post-migration.py", line 36, in migrate
    migrate_inventory_revaluation(env)
  File "/opt/odoo9/parts/stock-logistics-warehouse9/stock_inventory_revaluation/migrations/9.0.1.0.0/post-migration.py", line 29, in migrate_inventory_revaluation
    prod_dict = {'product_id': product_variants[0]}
IndexError: list index out of range`

I think that you should add:
`if product_variants:
    prod_dict = {'product_id': product_variants[0]}
    revaluation.write(prod_dict)
`